lib/Entities/NewsletterLinkEntity.php

```diff
@@ -22,20 +22,20 @@ class NewsletterLinkEntity {
   /**
    * @ORM\ManyToOne(targetEntity="MailPoet\Entities\NewsletterEntity")
    * @ORM\JoinColumn(name="newsletter_id", referencedColumnName="id")
-   * @var NewsletterEntity|null
+   * @var NewsletterEntity
    */
   private $newsletter;
 
   /**
    * @ORM\ManyToOne(targetEntity="MailPoet\Entities\SendingQueueEntity")
    * @ORM\JoinColumn(name="queue_id", referencedColumnName="id")
-   * @var SendingQueueEntity|null
```

Comment:
Those two still can be `null` due to possible data inconsistencies (`safelyLoadToOneAssociation`). Not sure if PHPStan complains if you keep the `|null`. I guess without foreign keys we'll always have this problem 😞 

https://github.com/mailpoet/mailpoet/blob/master/lib/Doctrine/EntityTraits/SafeToOneAssociationLoadTrait.php#L27